concent_api/core/subtask_helpers.py

```diff
@@ -277,7 +279,7 @@ def finalize_deposit_claim(
     ethereum_address: str,
     public_key: str,
 ) -> None:
-    deposit_claim = get_one_or_none(
+    deposit_claim: DepositClaim = get_one_or_none(  # type: ignore
```

Comment:
something's wrong here. `DepositClaim` and `# type: ignore` at the same time?

Reply:
see `get_one_or_none` return type. It returns Optional[Subtask]. This function was created to be generic, for different objects, not only subtask. Signature was changed in the meantime (I don't think it was good change), but function still works for different objects. 

In this case `DepositClaim` is passed as Model, so its instance is return type. I added `: DepositClaim` to use real/correct type, but ignoring type was also necessary. 

BTW.  typing in `get_one_or_none` should be fixed. What do you think?

Reply:
`get_one_or_none` definitely should get fixed. It was meant to work with any model.